### DISCUSSION TITLE
Forgot your username or password?

### DISCUSSION THREAD

**Discussion by bassbiteye** [ovh/manager#6479] [General]
hello guys i forgot my password but when i try to renit it, it told me An email has just been sent to you but i will not receive the email since 5th/02/2022
![ovh](https://user-images.githubusercontent.com/49490105/153907058-2c75151d-973e-4ac5-8269-c91009ba8065.PNG)
please help me it's very urgent

**Comment by antleblanc** [comment]
Hi @bassbiteye!

Unfortunately there's nothing we can do regarding your customer account.
Please contact our support team who will be able to help you or refer to https://help.ovhcloud.com.

Thank you!
